appveyor.yml

```diff
@@ -73,28 +78,8 @@ artifacts:
   - path: '*.Rcheck\**\*.Rout'
     name: Logs
 
-  - path: "ospsuite_*_ubuntu18.tar.gz"
-    name: Ubuntu18
-
-  - path: "ospsuite_*_centOS7.tar.gz"
-    name: CentOS7
+  - path: '\*_*.tar.gz'
+    name: Bits
```

Comment:
bits? what about you call in linux and the other one win ?

Reply:
Changed to the same as in _ospsuite_.